Dockerfile

```diff
@@ -49,5 +49,8 @@ RUN apt-get update && apt-get install -y \
 USER topos:topos
 
 RUN mkdir /tmp/shared
+RUN mkdir -p /tmp/node_config/subnet/topos
+RUN mkdir -p /tmp/node_config/node/test/consensus
+RUN mkdir -p /tmp/node_config/node/test/libp2p
```

Comment:
I tried, and no, I need to create the folders or it won't have it during the `init.sh` script and it fails.

Reply:
Then just create the topos home in the home folder and don't recreate the whole directory structure. I would prefer not to have complete tree in docker as it can change.